.pre-commit-config.yaml

```diff
@@ -26,6 +26,5 @@ repos:
     - id: staker-rewards-lint-js
       name: 'staker rewards lint js'
       entry: /usr/bin/env bash -c "cd staker-rewards && npm run lint:js"
-      files: '\.js$'
```

Comment:
We should leave the `files` filter here and maybe add `staker-rewards` directory to the pattern?

Reply:
Sure, done in 46324f9